android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/translate/TranslationError.kt

```diff
@@ -72,6 +72,14 @@ sealed class TranslationError(
     class CouldNotRestoreError(override val cause: Throwable?) :
         TranslationError(errorName = "could-not-restore", displayError = false, cause = cause)
 
+    /**
+     * Could not determine translation download size between two pairs.
+     *
+     * @param cause The original [Throwable] before it was converted into this error state.
+     */
+    class CouldNotDetermineSizeError(override val cause: Throwable?) :
```

Comment:
Should this be `CouldNotDetermineDownloadSizeError`

Reply:
Thanks, updated, much clearer name!